xbmc/cores/IPlayer.h

```diff
@@ -159,9 +159,6 @@ enum EINTERLACEMETHOD
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL=14,
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF=15,
   VS_INTERLACEMETHOD_DEINTERLACE_HALF=16,
-  VS_INTERLACEMETHOD_DXVA_BOB = 17,
```

Comment:
ok, go ahead, I was just wondering. imo a safeguard does not make much sense if gaps are allowed. but this a different story and not related to this pr.